IBPSA/BoundaryConditions/WeatherData/ReaderTMY3.mo

```diff
@@ -1452,9 +1460,28 @@ and allows the following configurations:
   </td>
 </tr>
 </table>
+<!-- ============================================== -->
+<h4>Length of weather data and simulation period</h4>
+<p>
+If weather data span a year, which is the default for TMY3 data, or multiple years,
+then this model can be used for simulations that span multiple years. The simulation
+start time need to be set to the clock time of the respective start time. For example,
+to start at January 2 at 10am, set start time to <code>t=(24+10)*3600</code> seconds.
```

Comment:
I don't think this is sufficiently specific/clear. This assumes that the timezone that you define in the TMY3 file corresponds to the time zone of the date that you mention in the documentation?

Reply:
@Mathadon : Not sure what you mean by not being clear. Can you add/propose better wording? Note that the weather files for Chicago and San Francisco both start at `0`. Hence I am not sure what you mean.